### ISSUE TITLE
Label Encoding for fact verifacation over TabFact

### ISSUE THREAD

**Issue by vikxoxo** [microsoft/Table-Pretraining#12]
The Trained Tapex large model (especially availaible at Huggingface) has 0 label corresponding to Entailed class or 0 label corresponding to Refute class?
As by default, the TabFact when loaded through huggingface dataset has 0 corresponding to Refute class.
Please tell us this.

**Comment by SivilTaram** [comment]
@vikxoxo Thanks for your interest on our work! `0` corresponds to `Refute` and `1` corresponds to `Entailed`, following the same with TabFact.

**Comment by SivilTaram** [comment]
@vikxoxo Sorry for the previous message. I have just realized that the output is reversed, i.e., the current output of `tapex-large-finetuned-tabfact` means that `0` corresponds to `Entailed`.

**Comment by SivilTaram** [comment]
@vikxoxo Hi, I have created the official `tapex-large-finetuned-tabafct` under the `microsoft` namespace and its label space is correct! Try it if you're interested. :)
